tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-7-1-ci-pipeline.yml

```diff
@@ -0,0 +1,18 @@
+jobs:
+- job: Linux_CI_GPU_TENSORRT_Dev
+  pool: onnxruntime-tensorrt-linuxbuild 
```

Comment:
Like CUDA 10.2, can you put it in https://aiinfra.visualstudio.com/Lotus and make it run daily?

Reply:
Will do.